include/crm/common/results.h

```diff
@@ -50,6 +50,7 @@ extern "C" {
 #  define pcmk_err_multiple             213
 #  define pcmk_err_node_unknown         214
 #  define pcmk_err_already              215
+#  define pcmk_err_bad_option           216
```

Comment:
When adding a new option, we have some ancillary helpers that need updating as well: see lib/common/results.c pcmk_errorname() and pcmk_strerror().

Also, many command-line tools exit with a status of crm_errno2exit(). This allows them to just pass an rc from a function and get a reasonable exit status. Any return code that doesn't have something more specific there will get CRM_EX_ERROR (1). In this case, CRM_EX_INVALID_PARAM, CRM_EX_USAGE, or CRM_EX_DATAERR might be appropriate. In some cases, you might even want a new exit code to go along with it (though not necessary here; that would mainly be if a unique code was needed for scripts to test, or a clearer exit message was needed).

Reply:
I'm thinking maybe bad_nvpair would be clearer than bad_option (which could also be interpreted as a command-line option)